### PR TITLE
feat(buidmaster,census,pkgrepo,updatesite,usage) apply error log rotation on apache2

### PR DESCRIPTION
**Pull request by dduportal**
Ref. https://github.com/jenkins-infra/helpdesk/issues/2969

As per https://forge.puppet.com/modules/puppetlabs/apache/3.5.0#error_log_pipe.

### CODE REVIEW

File: dist/profile/manifests/usage.pp [hunk 1]
```diff
@@ -164,7 +165,8 @@
     override        => ['All'],
     docroot         => $docroot,
     error_log_file  => "${usage_fqdn}/error_nonssl.log",
-    access_log_pipe => "|/usr/bin/rotatelogs ${apache_log_dir}/access_${::ipaddress}_nonssl.log.%Y%m%d%H%M%S 86400",
+    access_log_pipe => "|/usr/bin/rotatelogs ${apache_log_dir}/access_${facts['facts['networking']['ip']']}_nonssl.log.%Y%m%d%H%M%S 86400",
```
Comment (lemeurherve) [inline dist/profile/manifests/usage.pp]: Why this one is different?

Comment (dduportal) [reply dist/profile/manifests/usage.pp]: oh, sounds like it is the puppet autoformatting in my browser. Lemme revert

Comment (dduportal) [reply dist/profile/manifests/usage.pp]: fixed. thanks for the review!

**Review by lemeurherve — APPROVED** [review]
👍

**Comment by dduportal** [comment]
It's friday morning: let's break things in production :trollface:

**Pull request merged**